test/skate/detours/missed_stops_test.exs

```diff
@@ -200,18 +201,267 @@ defmodule Skate.Detours.MissedStopsTest do
       # North Quincy
       connection_end_index = 53
 
-      param = %Skate.Detours.MissedStops{
+      param = %MissedStops{
         shape: shape,
         stops: stops,
         connection_start: Enum.at(stops, connection_start_index),
         connection_end: Enum.at(stops, connection_end_index)
       }
 
       assert %MissedStops.Result{missed_stops: missed_stops} =
-               Skate.Detours.MissedStops.missed_stops(param)
+               MissedStops.missed_stops(param)
 
       assert Enum.slice(stops, connection_start_index..(connection_end_index - 1)) ==
                missed_stops
     end
+
+    test "counts stops as missed if they're just inside the detour" do
+      #                               Both stops are missed
```

Comment:
Normally I prefer Github's split-diff view, but it really messes with this otherwise truly beautiful ASCII-art 😂